### PR TITLE
Add attribute value next DrillToURL label 

### PR DESCRIPTION
**Pull request by ivanmjartan**
- Port TNT-83 Extend D2Uri picker item

JIRA: RAIL-3597

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4741/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6373/ : SUCCESS in 17m 35s

File: libs/sdk-ui-dashboard/src/presentation/drill/utils/drillDownUtils.ts [hunk 1]
```diff
@@ -3,23 +3,32 @@
...
+                intersectionAttributeItem.attributeHeader.localIdentifier === localIdentifier,
         )
         .map((intersectionAttributeItem) => intersectionAttributeItem.attributeHeaderItem.name)[0];
 }
 
+/**
+ * Get total number of IDrillToUrl
+ * @internal
+ */
+export function getTotalDrillsToUrl(drillDefinition: DashboardDrillDefinition[]): number {
```
Comment (no23reason) [inline libs/sdk-ui-dashboard/src/presentation/drill/utils/drillDownUtils.ts:28]: The name is a bit strange, what about `getTotalDrillToUrlCount`?

Comment (ivanmjartan) [reply libs/sdk-ui-dashboard/src/presentation/drill/utils/drillDownUtils.ts]: Fixed

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1605

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1341/ : SUCCESS in 1m 11s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1617/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/396/ : SUCCESS in 13m 46s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6379/ : SUCCESS in 21m 57s

**Pull request merged**